primitives/io/src/lib.rs

```diff
@@ -427,6 +427,24 @@ pub trait Trie {
 	fn keccak_256_ordered_root(input: Vec<Vec<u8>>) -> H256 {
 		Layout::<sp_core::KeccakHasher>::ordered_trie_root(input)
 	}
+
+	/// Verify trie proof
+	fn blake2_256_verify_proof(root: H256, proof: &[Vec<u8>], key: &[u8], value: &[u8]) -> bool {
```

Comment:
Maybe:
```suggestion
	fn blake2_256_verify_proof(root: H256, proof: &[Vec<u8>], key: &[&[u8]], value: &[&[u8]]) -> bool {
```

Reply:
That's not supported by the `runtime_interface` macro unfortunately